curvefs/src/client/kvclient/kvclient_manager.cpp

```diff
@@ -52,29 +61,51 @@ void KVClientManager::Uninit() {
 
 void KVClientManager::Set(std::shared_ptr<SetKVCacheTask> task) {
     threadPool_.Enqueue([task, this]() {
-        LatencyGuard guard(&kvClientMetric_.kvClientSet.latency);
+        LatencyGuard guard(&kvClientManagerMetric_->set.latency);
 
         std::string error_log;
         task->res =
             client_->Set(task->key, task->value, task->length, &error_log);
-        task->timer.stop();
-        ONRETURN(Set, task->res);
-
-        task->done(task);
+        if (task->res) {
+            kvClientManagerMetric_->count << 1;
+        }
+        OnReturn(&kvClientManagerMetric_->set, task);
     });
 }
 
+void UpdateHitMissMetric(memcached_return_t retCode,
+                         std::shared_ptr<KVClientManagerMetric> metric) {
+    // https://awesomized.github.io/libmemcached/libmemcached/memcached_return_t.html
+    switch (retCode) {
+        case MEMCACHED_SUCCESS:
+            metric->hit << 1;
+            break;
+        case MEMCACHED_DATA_DOES_NOT_EXIST:
+            // The data requested with the key given was not found.
+        case MEMCACHED_DATA_EXISTS:
```

Comment:
same meaning of exists and not exist?

Reply:
> same meaning of exists and not exist?

![image](https://github.com/opencurve/curve/assets/15775037/31bae4ac-f3b9-4baf-85d7-df261ca3b4c6)

Reply:
it's interesting ⁉️